psychopy/tools/pkgtools.py

```diff
@@ -240,27 +240,24 @@ def installPackage(
             and error from the subprocess.
         If `awaited=False`:
             Returns the job (thread) which is running the install.
-    """
-    if target is None:
-        target = prefs.paths['userPackages']
-    
+    """   
     # convert extra to dict
     if extra is None:
         extra = {}
-    # check the directory exists before installing
-    if not os.path.exists(target):
-        raise NotADirectoryError(
-            'Cannot install package "{}" to "{}", directory does not '
-            'exist.'.format(package, target))
+
 
     # construct the pip command and execute as a subprocess
     cmd = [sys.executable, "-m", "pip", "install", package]
 
     # optional args
     if target is None:  # default to user packages dir
-        cmd.append('--prefix')
```

Comment:
This then tries to use default user packages dir as `--prefix`, which is a different behavior from using `--user`. I think the correct order of checking should use another if. It's easier for me to make a PR to show what I was thinking.

Reply:
We configure the package directory as the user site since we can just pass that down as an environment variable to the subprocess.

Reply:
Are you referring to the `[PYTHONUSERBASE](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUSERBASE)` environment variable? I totally agree with using that and `--user` as the only way to manage pip installation destinations, but if so what's the reason for storing both `prefs.paths['userPackages']` and `prefs.paths['packages']` in the preference setting, and using the `--target` flag? Can users change these in preferences?